conda_recipes/maya-2025/recipe/build.sh

```diff
@@ -27,7 +27,9 @@ ln -r -s "$INSTALL_DIR/bin/maya$MAYA_VERSION" "$INSTALL_DIR/bin/maya"
 # from the system package manager, dnf.
 mkdir -p "$SRC_DIR/download"
 cd "$SRC_DIR/download"
-dnf download --resolve -y freetype alsa-lib fontconfig harfbuzz libbrotli graphite2 libxkbfile
+dnf download --resolve -y freetype alsa-lib fontconfig harfbuzz libbrotli graphite2 \
```

Comment:
Have we asked autodesk  why more and more dependencies are needed? You are adding X windows dependencies, which gets very heavy in the long run.

Reply:
If every DCC now starts to need these X binaries, it is much better to just put it in to the base AMI.

Reply:
- I believe this is need to in the case of V-Ray rather than Maya (Maya will work without this but V-Ray for Maya will not work).

- We could add this into V-Ray conda recipe and link Maya to it, but this pose some potential error: Modifier could easily change path inside conda recipe build script and it will break V-Ray for Maya.

- What we could do is document this and see if only V-Ray need this or lots of DCC need it and make a decision to add it to the base AMI or not.

Reply:
Yes - document is the key here so we can see what patterns of dependencies are needed in the SMF worker image vs conda recipie.

The more we put into the conda package, the more vulnerabilities we have to patch by hand

Reply:
Could we include the additional ones in the v-ray package? It's just those binaries with the dependency, right?

Reply:
The Maya .so files should be fine, we  just have to apply that to the vray ones. What were you finding that made it different?

Reply:
The think that's make it diffrent is when invoke Maya, it was able to run. But then Maya need some dependencies to be run to call V-Ray, which in this case it's not runable due to missing dependencies.

By adding these package to V-Ray conda recipe rather than here, we need to link a specific .so file from Maya (not Maya or Render .so file) to a lib we store those downloaded dependencies, which seem unstable in my opinion. 

Also since all those packages only needed for Maya and have nothing to do with V-Ray (V-Ray executable do not need it), leaving it in V-Ray seem like a not idea choice

Reply:
Thanks for updating the description with those additional details, that helps understand the chang.e